frontend/ts/src/ServiceApi.ts

```diff
@@ -643,3 +644,23 @@ async function ledgerService(identity: Identity): Promise<LedgerService> {
 
   return ledgerBuilder(agent);
 }
+
+// Exports methods that can be triggered from the console.
+// Eventually we'll add support to all the commands, but that's better and easier
+// to do once we migrate to using @dfinity/nns.
+//
+// e.g. await nns.governance.addHotKey("<neuron-id>", "<principal>")
+function setupExports(governance: GovernanceService) {
+  // @ts-ignore
+  window["nns"] = {
+    governance: {
+      // Modifying the method signature to be a bit more ergonomic for the console.
+      addHotKey: function (neuronId: NeuronId, principal: PrincipalString) {
+        return governance.addHotKey({
+          neuronId: neuronId,
+          principal: principal,
+        });
```

Comment:
Presumably this can be made more compact:
```suggestion
        return governance.addHotKey({ neuronId, principal });
```
But this is not important.  If ci is green, please press merge!

Reply:
Prettier doesn't seem to like that unfortunately.